test/bin/scenario.sh

```diff
@@ -38,6 +38,38 @@ full_vm_name() {
     echo "${SCENARIO//@/-}-${base}"
 }
 
+# hostname validation
+# based on https://github.com/rhinstaller/anaconda/blob/c95142f76735a2e9ae6d845f8569d46632ddd619/pyanaconda/network.py#L96-L120
+validate_vm_hostname() {
+    local vm_name="$1"
+
+    if [ -z "${vm_name}" ]; then
+        error "VM hostname cannot be empty string" 
+        record_junit "${vm_name}" "vm_hostname_validation" "FAILED"
+        exit 1
+    fi
+
+    if [ ${#vm_name} -gt 64 ]; then
+        error "VM hostname is too long" 
+        record_junit "${vm_name}" "vm_hostname_validation" "FAILED"
+        exit 1
+    fi
+
+    if [[ "${vm_name}" =~ ^\. ]]; then
+        error "VM hostname cannot start with '.'" 
+        record_junit "${vm_name}" "vm_hostname_validation" "FAILED"
+        exit 1
+    fi
```

Comment:
Do we need this check? I mean if the name starts from `.`, wouldn't the next check fail?

Reply:
Yes, however this check has explicit error message about what is wrong

Reply:
On the other hand: before passing the hostname to this function, we replace all `.` with `-`, so this won't be executed unless we do some refactoring and mess up

Reply:
Okay, I removed it